### ISSUE TITLE
How to reflash firmware

### ISSUE THREAD

**Issue by bortek** [ge0rg/samsung-nx-hacks#19]
Hi

I have modded the camera and after "systemctl rescue" it does not want to complete the bootup process. Green led continues to flash. I have been doing some mods and systemctl rescue for some time and did not have any issues. Not until now.

How do I reflash the firmware? Here in various places there is a mentioning about reflashing firmware if something goes wrong. I tried with i-Launcher on Win10 but it does not recognize the camera.

Is there any way to get into the recovery mode and start from there?

**Comment by ottokiksmaler** [comment]
Answered in another issue.
